dynamic-property-polling/src/test/kotlin/ru/fix/dynamic/property/api/test/PollingPropertyTest.kt

```diff
@@ -26,7 +26,7 @@ class DynamicPropertyTest {
         val scheduler = mockk<ReschedulableScheduler>(relaxed = true)
         val pollingTask = slot<Runnable>()
         every {
-            scheduler.schedule(any(), any(), capture(pollingTask))
```

Comment:
in 3.0.1 stdlib there are two methods matching this signature, so i had to specify type of second arg